lib/modules/datasource/crate/index.ts

```diff
@@ -112,6 +112,9 @@ export class CrateDatasource extends Datasource {
         const release: Release = {
           version: version.vers.replace(/\+.*$/, ''),
         };
+        if (release.version !== version.vers) {
+          release.versionOrig = version.vers;
```

Comment:
@rarkins I think it should be stored in a property like the managerData instead because this field can be overwritten by a user extract config 🤔